apps/sasquatch/src/androidTest/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivityTest.java

```diff
@@ -0,0 +1,203 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
This test must check with MobileCenter isEnabled calls to be meaningful for Mobile Center testing.